.github/CHANGELOG.md

```diff
@@ -202,55 +252,55 @@ Oluwatobi Ogunbayo, Maria Schuld, Antal Száva.
 
 # Release 0.6.1
 
-### New features since last release
+<h3>New features since last release</h3>
 
 * Added a `print_applied` method to QNodes, allowing the operation
   and observable queue to be printed as last constructed.
-  [#378](https://github.com/XanaduAI/pennylane/pull/378)
+  [(#378)](https://github.com/XanaduAI/pennylane/pull/378)
 
-### Improvements
+<h3>Improvements</h3>
 
 * A new `Operator` base class is introduced, which is inherited by both the
   `Observable` class and the `Operation` class.
-  [#355](https://github.com/XanaduAI/pennylane/pull/355)
+  [(#355)](https://github.com/XanaduAI/pennylane/pull/355)
 
 * Removed deprecated `@abstractproperty` decorators
   in `_device.py`.
-  [#374](https://github.com/XanaduAI/pennylane/pull/374)
+  [(#374)](https://github.com/XanaduAI/pennylane/pull/374)
 
 * The `CircuitGraph` class is updated to deal with `Operation` instances directly.
-  [#344](https://github.com/XanaduAI/pennylane/pull/344)
+  [(#344)](https://github.com/XanaduAI/pennylane/pull/344)
 
 * Comprehensive gradient tests have been added for the interfaces.
-  [#381](https://github.com/XanaduAI/pennylane/pull/381
+  [(#381)](https://github.com/XanaduAI/pennylane/pull/381
```

Comment:
```suggestion
  [(#381)](https://github.com/XanaduAI/pennylane/pull/381)
```

Reply:
Oh, good catch. I used regex, it obviously wasn't a good enough match!